server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/PreviewDTO.java

```diff
@@ -0,0 +1,66 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.draft.model;
+
+import java.util.Objects;
+
+import org.apache.james.jmap.api.preview.Preview;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.base.Preconditions;
+
+public class PreviewDTO {
+
+    public static PreviewDTO from(Preview preview) {
+        return new PreviewDTO(preview.getValue());
+    }
+
+    public static PreviewDTO of(String value) {
```

Comment:
what is the use-case for this one? I guess Preview are always generated by the server and sent to the client. Thus, it's always a Preview object when it reach the protocol project.

Reply:
only used in tests (MessageFullViewFactoryTest)